### PR TITLE
Sorting redo

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

File: content/2_Bronze/Intro_Sorting.mdx [hunk 1]
```diff
@@ -161,21 +157,19 @@ $\Theta(N^2)$ behavior in quicksort.
 See [here](https://codeforces.com/contest/1324/hacks/625031/) for an example of
 a solution that was hacked on CodeForces.
 
-Two ways to avoid this:
+There's two ways to avoid this:
```
Comment (bqi343) [inline content/2_Bronze/Intro_Sorting.mdx:160]: There are?

Comment (SansPapyrus683) [reply content/2_Bronze/Intro_Sorting.mdx:160]: that was what the og said
i'm just changing the grammar

Comment (bqi343) [reply content/2_Bronze/Intro_Sorting.mdx:160]: "There's two ways" is incorrect

Comment (SansPapyrus683) [reply content/2_Bronze/Intro_Sorting.mdx:160]:
```suggestion
There are two ways
```

**Review by bqi343 — APPROVED** [review]

**Pull request merged**